docs/src/postgresql_conf.md

```diff
@@ -317,6 +317,15 @@ host all all all scram-sha-256 # (or md5 for PostgreSQL version <= 13)
 
 Refer to the PostgreSQL documentation for [more information on `pg_hba.conf`](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html).
 
+Inside the cluster manifest, `pg_hba` lines are added as list items
+in `spec.postgresql.pg_hba`. e.g.:
+
+``` yaml
+  postgresql:
+    pg_hba:
+      - host all all 10.244.0.0/16 md5
```

Comment:
```suggestion
      - hostssl app app 10.244.0.0/16 md5
```

Reply:
I would rather encourage usage of SSL in examples. Also I am restricting access to the app database for the app user, using md5.